zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/IPAuthenticationProvider.java

```diff
@@ -55,9 +62,16 @@ public List<Id> handleAuthentication(HttpServletRequest request, byte[] authData
     // This is a bit weird but we need to return the address and the number of
     // bytes (to distinguish between IPv4 and IPv6
     private byte[] addr2Bytes(String addr) {
-        byte[] b = v4addr2Bytes(addr);
-        // TODO Write the v6addr2Bytes
-        return b;
+        if (addr.contains(":")) {
+            LOG.info("Attempting to parse as IPv6...");
```

Comment:
Please don't log these messages at info level. We're not interested at all. Actually there's no additional information in this log message, so you can just remove it completely.

You might want to add a debug level message in case of a successful parsing:
```
LOG.debug("Address was parsed successfully as V4/V6 address: {}", addr);
```
but I don't think we need that either.